azuredevops/internal/acceptancetests/resource_serviceendpoint_aws_terraform_test.go

```diff
@@ -0,0 +1,179 @@
+					testutils.CheckServiceEndpointExistsWithName(tfSvcEpNode, serviceEndpointNameFirst), resource.TestCheckResourceAttrSet(tfSvcEpNode, "project_id"),
+					resource.TestCheckResourceAttr(tfSvcEpNode, "service_endpoint_name", serviceEndpointNameFirst),
+				),
+			},
+			{
+				Config: hclSvcEndpointAwsTerraformResourceUpdate(projectName, serviceEndpointNameSecond, description),
+				Check: resource.ComposeTestCheckFunc(
+					testutils.CheckServiceEndpointExistsWithName(tfSvcEpNode, serviceEndpointNameSecond),
+					resource.TestCheckResourceAttrSet(tfSvcEpNode, "project_id"),
+					resource.TestCheckResourceAttr(tfSvcEpNode, "service_endpoint_name", serviceEndpointNameSecond),
+					resource.TestCheckResourceAttr(tfSvcEpNode, "description", description),
+				),
+			},
+		},
+	})
+}
+
+func TestAccServiceEndpointAwsTerraform_requiresImportErrorStep(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+	serviceEndpointName := testutils.GenerateResourceName()
+
+	resourceType := "azuredevops_serviceendpoint_aws_terraform"
+	tfSvcEpNode := resourceType + ".test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testutils.PreCheck(t, nil) },
+		Providers:    testutils.GetProviders(),
+		CheckDestroy: testutils.CheckServiceEndpointDestroyed(resourceType),
+		Steps: []resource.TestStep{
+			{
+				Config: hclSvcEndpointAwsTerraformResource(projectName, serviceEndpointName),
+				Check: resource.ComposeTestCheckFunc(
+					testutils.CheckServiceEndpointExistsWithName(tfSvcEpNode, serviceEndpointName),
+					resource.TestCheckResourceAttrSet(tfSvcEpNode, "project_id"),
+					resource.TestCheckResourceAttr(tfSvcEpNode, "service_endpoint_name", serviceEndpointName),
+				),
+			},
+			{
+				Config:      hclSvcEndpointAwsTerraformResourceRequiresImport(projectName, serviceEndpointName),
+				ExpectError: testutils.RequiresImportError(serviceEndpointName),
+			},
+		},
+	})
+}
+
+func hclSvcEndpointAwsTerraformResource(projectName string, serviceEndpointName string) string {
+	return hclSvcEndpointAwsTerraformResourceUpdate(projectName, serviceEndpointName, "description")
+}
+
+func hclSvcEndpointAwsTerraformResourceUpdate(projectName string, serviceEndpointName string, description string) string {
+	serviceEndpointResource := fmt.Sprintf(`
+	resource "azuredevops_serviceendpoint_aws_terraform" "test" {
+		project_id             = azuredevops_project.project.id
+		access_key_id          = "0000"
+		secret_access_key      = "secretkey"
+		region                 = "us-east-1"
+		service_endpoint_name  = "%s"
+		description            = "%s"
+	}`, serviceEndpointName, description)
+
+	projectResource := testutils.HclProjectResource(projectName)
+	return fmt.Sprintf("%s\n%s", projectResource, serviceEndpointResource)
+}
+
+func hclSvcEndpointAwsTerraformResourceComplete(projectName string, serviceEndpointName string, description string, sessionToken string, rta string, rsn string, externalId string) string {
```

Comment:
essionToken, rta s, rsn
```suggestion
func hclSvcEndpointAwsTerraformResourceComplete(projectName, serviceEndpointName, description string) string {
```